hal/targets/hal/TARGET_STM/TARGET_STM32F0/TARGET_NUCLEO_F091RC/device.h

```diff
@@ -43,6 +43,8 @@
 
 #define DEVICE_I2C              1
 #define DEVICE_I2CSLAVE         1
+#define DEVICE_I2C_ASYNCH       1
+#define DEVICE_I2C_ASYNCH_DMA   1
```

Comment:
Can you please rebase on top of master, to get features, and this would require to be updated then ?